windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -62,7 +62,7 @@ This release adds the following new features and improvements.
 [id="wmco-2-0-0-vsphere-support"]
 ==== Support for clusters running on VMware vSphere
 
-You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create a Windows MachineSet object on vSphere to host Windows Server compute nodes.
```

Comment:
on a cluster that is installed on VMware...

Suggestion to add that. Idk if its grammatically necessary 🤷‍♀️